apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -646,15 +646,33 @@ export default Vue.extend({
           editorParams: {
             verticalNavigation: useVerticalNavigation ? 'editor' : undefined,
             search: true,
-            values: /^(bool|boolean)$/i.test(column.dataType)
-              ? [true, false]
-              : undefined,
             allowEmpty: true,
             // elementAttributes: {
             //   maxLength: column.columnLength // TODO
             // }
           },
         }
+
+        if (/^(bool|boolean)$/i.test(column.dataType)) {
+          // BigInt to match the value from backend
+          const values = [
+            { label: '0', value: BigInt(0) },
```

Comment:
Also, for non-sqlite, shouldn't the values be 'true' and 'false' also?

Reply:
Oh, sorry I missed the `dropdown will show false/true` section below.

Question though -- do `0` and `1` map to boolean data types in all the other databases? Maybe that needs a test to confirm

Reply:
Oh I thought boolean is only in sqlite. 0 and 1 do not map to boolean types in postgres. With that I think it's simpler to use false / true like before. It's easy to switch. Give me a second.

Reply:
Actually, it needs some generic mapping like `dialect.booleanValues: { "true": BigInt(1), "false": "BigInt(0)" }`, and then do this to each database.

Reply:
The only thing that changes here is in sqlite, we're showing `0` and `1` in dropdown, instead of `false` and `true`. That's probably the simplest solution we could have. Is that ok?

The problem with sqlite is that boolean is represented as `0` and `1`, and when using `false` and `true` as the dropdown options, it doesn't match up, which then shows that `(empty)` value.

We still can use the `false` and `true` as values for sqlite though if you want. I just need some time to wire up the good solution to that.